src/aind_physiology_fip/add_session_timing.py

```diff
@@ -0,0 +1,88 @@
+"""Add session start and end times to session_input.json from camera metadata."""
+
+import json
+import sys
+from datetime import datetime
+from pathlib import Path
+from zoneinfo import ZoneInfo
+
+import pandas as pd
+
+
+def add_session_timing(data_dir: str, local_timezone: str = "America/Los_Angeles") -> None:
+    """
+    Add session_start_time and session_end_time to session_input.json.
+
+    Reads camera metadata CSV to extract first and last CpuTime values,
+    converts them to local timezone, and adds them to session_input.json.
+
+    Parameters
+    ----------
+    data_dir : str
```

Comment:
data_dir should be the root of the dataset so people dont have to have any additional knowedge of the underlying structure. In practice: `data_dir / fib / fip_xxxx/ Logs/ session_input.json`